hydroshare/settings.py

```diff
@@ -690,8 +690,18 @@
 # detect test mode to turn off some features
 TESTING = len(sys.argv) > 1 and sys.argv[1] == 'test'
 
+# HydroShare web services settings
+HSWS_URL = "http://10.37.142.23:8060/his/services/update"
+HSWS_API_TOKEN = "d1caabf3059ef7ffd99d890aba57100da2e0efc8"
```

Comment:
@kjlippold Should the api token be kept as a secret? If so, that need to go into local_settings.

Reply:
Yes, it should be secret in production. I moved these settings from local_settings to settings in response to https://github.com/hydroshare/hydroshare/pull/4023#discussion_r520169829. I can move them back to local_settings if that's where they should be. We do want to keep these settings together though.

Reply:
OK. Then they should go to local_settings then. I assume the url and token you put here in settings are not real secrets but for testing only or for getting local development and a development auto-build working. If that is indeed the case, you can leave them here, and define the real values in local_settings with the same variable names, which will override whatever are defined in settings.